sdks/python/apache_beam/examples/streaming_wordcount_it_test.py

```diff
@@ -81,7 +81,7 @@ def tearDown(self):
   @attr('IT')
   def test_streaming_wordcount_it(self):
     # Build expected dataset.
-    expected_msg = [('%d: 1' % num) for num in range(DEFAULT_INPUT_NUMBERS)]
+    expected_msg = [(b'%d: 1' % num) for num in range(DEFAULT_INPUT_NUMBERS)]
```

Comment:
I think we should instead decode `bytes` that we receive from PubSub in https://github.com/apache/beam/blob/79f8e34fc75c59297c0c7f07ce53325ce96bb5cf/sdks/python/apache_beam/io/gcp/tests/pubsub_matcher.py#L106.

The matcher expects strings:
https://github.com/apache/beam/blob/79f8e34fc75c59297c0c7f07ce53325ce96bb5cf/sdks/python/apache_beam/io/gcp/tests/pubsub_matcher.py#L58

PubSub client returns bytes: https://github.com/apache/beam/blob/a5ed104aeac91781492523808fa4d3df6f2e55f7/sdks/python/apache_beam/io/gcp/pubsub.py#L60

Given this is a test code we can assume a 'utf-8' encoding

Reply:
Ok thank you! I changed it